src/migrations/20240715000000-fix-old-session-regions.js

```diff
@@ -0,0 +1,45 @@
+const {
+  prepMigration,
+} = require('../lib/migration');
+
+module.exports = {
+  up: async (queryInterface) => queryInterface.sequelize.transaction(
+    async (transaction) => {
+      await prepMigration(queryInterface, transaction, __filename);
+      await queryInterface.sequelize.query(/* sql */`
+
+        -- One very old session lacks the regionId value
+        -- This finds and sets it
+        DROP TABLE IF EXISTS sr_updates;
+        CREATE TEMP TABLE sr_updates
+        AS
+        WITH updater AS (
+        UPDATE "SessionReportPilots" srp
+        SET data = JSONB_SET(srp.data,'{regionId}',TO_JSONB(erp."regionId"))
+        FROM "EventReportPilots" erp
+        WHERE erp.id = srp."eventId"
+          AND srp.data->>'regionId' = ''
```

Comment:
Would this also catch cases if regionId is null in the jsonb not sure if its a valid case.

Reply:
Ah didnt realize there was only 1 session nvm.